### PR TITLE
Remove md shortcuts and their references

### PR DESCRIPTION
**Pull request by carlossanlop**
From a conversation with @gewarren, we want to remove the *.md shortcut files from the ~/include folder. These files contain texts representing official ways of calling different products or services, and they are referenced in markdown remarks all over dotnet-api-docs xmls.

We need to remove them because they prevent the [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool) from converting many remarks from CDATA+markdown into plain text (there is no way to convert INCLUDE sections to xml). We prefer to use plain xml so that Roslyn can validate triple slash comments (it is unable to validate a CDATA section). 

I am requesting help from the heavy cavalry to review this huge PR: @gewarren @BillWagner @dotnet/docs @mairaw.

The PR is split in individual commits for each shortcut file to make it easier to review. I also deleted the referenced files, plus a few others that had no references anywhere.

I limited the removal of *.md files to only those whose contents have less than 80 characters. All other larger files can stay because they are what we call "lengthy remarks" and their use is repeated in many places.

In case anyone is curious, I wrote a tool to automate the substitution of all these shortcut files and uploaded it here: https://github.com/carlossanlop/experiments/tree/ShortcutReplacer

cc @jeffhandley

### CODE REVIEW

**Review by gewarren — APPROVED** [review]
This is awesome. Thanks @carlossanlop! For the `XAML2009` replacement, just wondering if there should be a space in the middle.

**Comment by carlossanlop** [comment]
I would not have done it if you had not suggested it, @gewarren :)

You're talking about `MSBuild`? I have always known it like that, without space in the middle.

File: xml/System.Xaml/XamlLanguage.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A factory method is a [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)] feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.  
+ A factory method is a XAML2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.
```
Comment (gewarren) [inline xml/System.Xaml/XamlLanguage.xml:103]:
```suggestion
 A factory method is a XAML 2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.  
```

File: xml/System.Xaml/XamlLanguage.xml [hunk 2]
```diff
@@ -160,7 +160,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `AsyncRecords` concept is part of [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)]. See [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).  
+ The `AsyncRecords` concept is part of XAML2009. See [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).
```
Comment (gewarren) [inline xml/System.Xaml/XamlLanguage.xml:163]:
```suggestion
 The `AsyncRecords` concept is part of XAML 2009. For more information, see [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).  
```

File: xml/System.Xaml/XamlLanguage.xml [hunk 3]
```diff
@@ -334,7 +334,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This directive is part of [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)]. See [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).  
+ This directive is part of XAML2009. See [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).
```
Comment (gewarren) [inline xml/System.Xaml/XamlLanguage.xml:337]:
```suggestion
 This directive is part of XAML 2009. For more information, see [XAML 2009 Language Features](/dotnet/framework/xaml-services/xaml-2009-language-features).  
```

File: xml/System.Xaml/XamlLanguage.xml [hunk 4]
```diff
@@ -501,7 +501,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A factory method is a [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)] feature for identifying an alternative technique that constructs the underlying type for a XAML type. The default technique, if a factory method is unavailable, is to try to find a parameterless constructor, and then try to find an identified type converter on type, member, or destination type.  
+ A factory method is a XAML2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The default technique, if a factory method is unavailable, is to try to find a parameterless constructor, and then try to find an identified type converter on type, member, or destination type.
```
Comment (gewarren) [inline xml/System.Xaml/XamlLanguage.xml:504]:
```suggestion
 A factory method is a XAML 2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The default technique, if a factory method is unavailable, is to try to find a parameterless constructor, and then try to find an identified type converter on type, member, or destination type.  
```

File: xml/System.Xaml/XamlLanguage.xml [hunk 5]
```diff
@@ -973,7 +973,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- `Reference` is a [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)] feature for referencing another object node in the markup. The implementation for .NET Framework XAML Services is found in <xref:System.Windows.Markup.Reference?displayProperty=nameWithType>.  
+ `Reference` is a XAML2009 feature for referencing another object node in the markup. The implementation for .NET Framework XAML Services is found in <xref:System.Windows.Markup.Reference?displayProperty=nameWithType>.
```
Comment (gewarren) [inline xml/System.Xaml/XamlLanguage.xml:976]:
```suggestion
 `Reference` is a XAML 2009 feature for referencing another object node in the markup. The implementation for .NET Framework XAML Services is found in <xref:System.Windows.Markup.Reference?displayProperty=nameWithType>.  
```

**Review by gewarren — COMMENTED** [review]
Here are the XAML2009 instances.

File: xml/System.Xaml/XamlLanguage.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Xaml/XamlLanguage.xml:103]: Oh, literally the word "XAML 2009". I searched for one of the md files with 2009 in their name and only found the MSBuild one.

**Comment by carlossanlop** [comment]
I got warnings but they do not seem to be related to my changes. @gewarren is it ok if we merge this?

**Pull request merged**